Jenkinsfile

```diff
@@ -698,6 +698,17 @@ core_logic: {
         }
       }
     },
+    'Scala: CentOS CPU': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/build-centos7-cpu') {
```

Comment:
Change the ws to something unique (ut-scala-centos7-cpu) and change utils.unpack_and_init from from the ubuntu based 'cpu' to 'centos7_cpu'